devise.rb

```diff
@@ -28,9 +27,11 @@
 run "unzip stylesheets.zip -d app/assets && rm -f stylesheets.zip && rm -f app/assets/rails-stylesheets-master/README.md"
 run "mv app/assets/rails-stylesheets-master app/assets/stylesheets"
 
+gsub_file("app/assets/stylesheets/application.scss", '@import "bootstrap/scss/bootstrap";', '@import "bootstrap";')
+
 inject_into_file "config/initializers/assets.rb", before: "# Precompile additional assets." do
   <<~RUBY
-    Rails.application.config.assets.paths << Rails.root.join("node_modules")
+    Rails.application.config.assets.paths << Rails.root.join("bootstrap.min.js popper.js")
```

Comment:
I feel like we don't actually need this ⚠️

Reply:
Nope still need to, that was a reminder to myself 😄

Reply:
Good news, it works without so I removed it from the script and corrected the `popper` package:

![image](https://user-images.githubusercontent.com/57643651/225299275-8ae7e86a-ca25-4696-8b0e-810f14e0abee.png)